### PR TITLE
Feature/add zmest landing

### PR DESCRIPTION
**Pull request by mickietrill**

### CODE REVIEW

File: src/pages/zmest/handleSendEmailClick.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  changeButtonStatus(response.status)
+}
+
+export const handleClick = e => {
+  e.preventDefault()
+  const name = document.getElementsByName('name').item(0).value
+  const email = document.getElementsByName('email').item(0).value
+  const message = document.getElementsByName('message').item(0).value
+
+  if (!emailWasSentRecently) {
```
Comment (Evoltic) [inline src/pages/zmest/handleSendEmailClick.js]: Where was an issue when a user can click multiple times and send multiple requests while the first request is in process. Is this behavior prevented?

File: src/pages/zmest/handleSendEmailClick.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    result = result.join('; ')
+  } else {
+    result = `Not Identified (doesn't exist)`
+  }
+
+  return result
+}
+
+const changeButtonStatus = status => {
+  const defaultButton = document
```
Comment (Evoltic) [inline src/pages/zmest/handleSendEmailClick.js]: Document doesn't exist in prerender process

**Review by Evoltic — CHANGES_REQUESTED** [review]

File: src/pages/zmest/handleSendEmailClick.js (hunk 1 above)
Comment (mickietrill) [reply src/pages/zmest/handleSendEmailClick.js]: Create a separate task for this problem. Will be resolved in a separate thread.

File: src/pages/zmest/handleSendEmailClick.js (hunk 2 above)
Comment (mickietrill) [reply src/pages/zmest/handleSendEmailClick.js]: done

**Review by Evoltic — APPROVED** [review]

File: src/pages/zmest/handleSendEmailClick.js (hunk 1 above)
Comment (Evoltic) [reply src/pages/zmest/handleSendEmailClick.js]: Done: https://app.asana.com/0/1109631325656644/1117628049122004/f

**Pull request merged**